projects/sandbox/src/app/dataset-content-summary/dataset-content-summary.component.scss

```diff
@@ -70,6 +65,27 @@
   }
 }
 
+.sort-asc,
+.sort-desc {
+  .sort-toggle.active {
+    position: relative;
+    color: $gray-light;
+
+    &.left::before,
+    &.right::after {
+      content: '\2304';
```

Comment:
suggestion same here to use a constant to indicate is an unicode arrow? &#2304;

Reply:
Done